testing/test_aux.out

```diff
@@ -7,8 +7,11 @@ Lint checks
 cmd/inbuild exit code 0
 Extended tests
 RESULT skip protocol.bacnet.version
-RESULT skip protocol.bacnet.version
-RESULT info protocol.bacnet.version Protocol version: 1
+RESULT skip protocol.bacnet.pic
+RESULT info protocol.bacnet.version
```

Comment:
@JoelRedstone Should this test not include the change that prints out the protocol version?

Reply:
When Francesco added the description after the result he forgot to remove a '**\n**' before the description itself. The description it's been printed but just a line below and it's not been picked up by Daq. Fixing it right now. I'm wondering how it's working fine on bacnet.version

Reply:
Do you mean that `Protocol version: 1` **shouldn't** be on the same line as `RESULT info protocol.bacnet.version`?

If so, I'm pretty sure they **should** be on the same line. @pisuke is this so?

Reply:
**Yes they need to be on the same line**.. That's what I was trying to say

Reply:
ahhh, I get you. Good spot!